exporter/signalfxexporter/translation/converter.go

```diff
@@ -416,6 +416,7 @@ func float64ToDimValue(f float64) string {
 // resource attributes, including a cloud host id (AWSUniqueId, gcp_id, etc.)
 // if it can be constructed from the provided metadata.
 func appendResourceAttributesToDimensions(dims []*sfxpb.Dimension, resourceAttr map[string]string) []*sfxpb.Dimension {
+	// TODO: Replace with internal/common/splunk/hostid.go once signalfxexporter is converted to pdata.
```

Comment:
Are you going to still remove the resource attributes used to derive the host id like the code below in this function does?  If so, it seems like there will need to be something more that the hostid module does to facilitate that instead of having to know the exact fields here.

Reply:
Nope, sapm won't remove them since for traces we want all those tags. I think signalfxexporter will probably need to maintain its own list of labels/dimensions to filter out. It doesn't need to be per-provider type like it is today, it can just delete all of them (provider, account id, region, zone, etc. regardless of provider type).